src/commands/bso/tames.ts

```diff
@@ -87,6 +87,12 @@ const feedableItems: FeedableItem[] = [
 		tameSpeciesCanBeFedThis: [TameType.Gatherer],
 		announcementString:
 			'Your tame can now collect items 10% faster thanks to the Ring of endurance helping them run for longer!'
+	},
+	{
+		item: getOSItem('Dwarven warhammer'),
+		description: '20% faster PvM',
```

Comment:
The Tame kills so slow already that I feel like 20% isn't really worth it for a dwarven warhammer.
Example: It takes 73 minutes for the tame to kill 2x QBD. 

I personally think DWWH should be at least 40% for the tame, also.

(the cost for DWWH is currently ~25b )